pkg/logging/logging.go

```diff
@@ -6,18 +6,21 @@ import (
 	"context"
 	"crypto/rand"
 	"encoding/hex"
+	"flag"
 	"fmt"
 	"log/slog"
 	"os"
 
 	"github.com/gin-gonic/gin"
 	"go.opentelemetry.io/otel/trace"
+	"k8s.io/klog/v2"
 )
 
 type contextKey string
 
 const (
 	loggingLevelEnvVar            = "LOG_LEVEL"
+	klogLevelEnvVar               = "KLOG_LEVEL"
```

Comment:
Should this name be consistent with `loggingLevelEnvVar`, i.e. `kLoggingLevelEnvVar`?